ddtrace/ext/aws.py

```diff
@@ -3,17 +3,18 @@
 from typing import TYPE_CHECKING
 from typing import Tuple
 
-from ddtrace.contrib.trace_utils import flatten_dict
+from ddtrace.contrib.trace_utils import set_flatten_tags
 
 
 if TYPE_CHECKING:
     from ddtrace.span import Span
 
 
-EXCLUDED_ENDPOINT = {"kms", "sts"}
+EXCLUDED_ENDPOINT = frozenset({"kms", "sts"})
 EXCLUDED_ENDPOINT_TAGS = {
-    "s3": {"params.Body"},
-    "firehose": {"params.Records"},
+    "s3": frozenset({"params.Body"}),
+    "firehose": frozenset({"params.Records"}),
+    "sqs": frozenset({"params.MessageBody"}),
```

Comment:
this seems like it should always be ignored, regardless of service?

Reply:
I would have thought that `MessageBody` is specific to the SQS service? But surely it makes sense to ignore `*Body` fields. Should we try and be "smarter" about this and take out anything that contains `Body`?

Reply:
I am not sure... it really might vary service by service, so this might be safest approach?

either that or we move to an allow list to be more selective /shrug